gateway/Cargo.toml

```diff
@@ -49,10 +56,10 @@ proc-macro-hack = { default-features = false, version = "0.5.7" }
 [features]
 default = ["compression", "rustls", "tracing", "flate2/zlib"]
 compression = ["flate2"]
-native = ["twilight-http/native", "twilight-gateway-queue/native", "tokio-tungstenite/native-tls"]
-rustls = ["rustls-native-roots"]
-rustls-native-roots = ["twilight-http/rustls-native-roots", "twilight-gateway-queue/rustls-native-roots", "tokio-tungstenite/rustls-tls-native-roots"]
-rustls-webpki-roots = ["twilight-http/rustls-webpki-roots", "twilight-gateway-queue/rustls-webpki-roots", "tokio-tungstenite/rustls-tls-webpki-roots"]
+native = ["native-tls", "twilight-http/native", "twilight-gateway-queue/native", "tokio-tungstenite/native-tls"]
+rustls = ["rustls-tls", "rustls-native-roots"]
```

Comment:
rustls is not supposed to be an actually used feature flag, it should remain an alias for `rustls-native-roots` until it will be removed in #1314. Use `cfg(any(feature = "rustls-native-roots", feature = "rustls-webpki-roots"))` instead of `cfg(feature = "rustls")`